pxr/imaging/plugin/hdRpr/rprApiAov.cpp

```diff
@@ -592,7 +592,7 @@ HdRprApiDepthAov::HdRprApiDepthAov(
                             vec4 alpha = ReadPixelTyped(alphaImage, coord.x, coord.y);
                             vec4 color = ReadPixelTyped(inputImage, coord.x, coord.y);
                             if (alpha.x == 0) {
-                                color = make_vec4(1.f, 1.f, 1.f, 1.f);
+                                color = make_vec4(0.99f, 0.99f, 0.99f, 0.99f);
```

Comment:
Could you put some comments here for the reason 0.99 is set?
For example, "depth = 1.f doesn't work to compose opacity with Houdini 20.5 or newer."

Also, if the value = 1.f doesn't work, should we also set "dstHi" parameter in `m_remapFilter to 0.99f?
https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/pull/733/files#diff-8a2cc46ce27bad8acdca816dbd2cac97944de1275242d0c8c890b9e741425f32R611

Reply:
Makes sense, thank you very much, added the comment!

Reply:
Thanks